### PR TITLE
feat: group members roles [AR-1041]

### PR DESCRIPTION
**Pull request by MohamadJaara**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1041" title="AR-1041" target="_blank">StoryAR-1041</a>  Group Admins
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

[AR-1041]

### Solutions

expose the member role from the Kalium and show the new info on the conversation member screen

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [x] I have added automated test to this contribution

#### How to Test

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

[AR-1041]: https://wearezeta.atlassian.net/browse/AR-1041?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-754/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-754/2/) **failed**.

File: app/src/main/kotlin/com/wire/android/di/CoreLogicModule.kt [hunk 1]
```diff
@@ -518,7 +519,7 @@ class UseCaseModule {
 
     @ViewModelScoped
     @Provides
-    fun getUserInfoUseCaseProvider(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId) =
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/di/CoreLogicModule.kt:521]: we kind of agreed to use this : provideGetUserInfoUseCase()

**Review by trOnk12 — APPROVED** [review]
plx check my one comment, also plx need time add screen shots for your awesome feature you are adding with a PR :P also a screenshot sometimes helps to spot some issues before the merge

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-754/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3857819)(3857819)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3857819)(3857819)-dev-release.apk)

**Pull request merged**